packages/components/pages/f-account-info/package.json

```diff
@@ -43,10 +43,11 @@
     "@justeat/f-globalisation": "1.0.0"
   },
   "peerDependencies": {
-    "@justeat/browserslist-config-fozzie": ">=1.2.0"
+    "@justeat/browserslist-config-fozzie": ">=1.2.0",
+    "vuex": ">=3.5.1"
```

Comment:
I think you [might also need this change](https://github.com/justeat/fozzie-components/pull/1540/files#diff-8d69a0c8f25e9aa68bf113b42dee7ed1f1dacb2c71b1a292101d02863e631608) to further prevent the packages from being bundled up when built?